src/Nox.Generator/Application/Factories/EntityFactory.template.cs

```diff
@@ -27,18 +29,18 @@
 
 namespace {{codeGeneratorState.ApplicationNameSpace}}.Factories;
 
-internal abstract class {{className}}Base : IEntityFactory<{{entity.Name}}Entity, {{entity.Name}}CreateDto, {{entity.Name}}UpdateDto>
+internal abstract class {{className}}Base : IEntityFactory<{{entity.Name}}Entity, {{entityCreateDto}}, {{entityUpdateDto}}>
```

Comment:
@rochar As we have only one factory per entity, it means we'll have to use UpsertDto instead of CreateDto and UpdateDto for Owned entities in all the commands. That is what we discussed, just wanted to confirm that at the end we can remove CreateDto and UpdateDto for owned entities.